### PR TITLE
Founder And Maintainers of CareerZunction fix issue

### PR DESCRIPTION
**Pull request by Ankitapanda25**
I have fixed issue the issue #23 of the component under docs in the website.

### CODE REVIEW

**Comment by Ankitapanda25** [comment]
@HimanshuNarware I have solved the issue that was assigned to me. Please do review it and give your feedback and also I hope you label it under GSSOC'24

**Comment by HimanshuNarware** [comment]
@Ankitapanda25 share a demo video

**Comment by Ankitapanda25** [comment]
@HimanshuNarware Here is the demo video, shows both the from the phone width and all the way to the desktop view.

[screen-capture.webm](https://github.com/HimanshuNarware/CareerZunction_Intern/assets/123666302/db9d2b9e-99bb-40c2-85e5-fea920c777d8)

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "react-dom": "^18.2.0",
     "react-icons": "^5.0.1",
     "react-redux": "^9.1.0",
+    "react-responsive-carousel": "^3.2.23",
```
Comment (HimanshuNarware) [inline package.json:15]: Don't push package.json files

**Review by HimanshuNarware — APPROVED** [review]

**Comment by HimanshuNarware** [comment]
@Ankitapanda25   feel free to create more issues also starred our repo

**Pull request merged**